api/enrich.py

```diff
@@ -0,0 +1,139 @@
+from functools import partial
+from uuid import uuid4
+
+import requests
+from datetime import datetime, timedelta
+from flask import Blueprint, current_app
+
+from api.schemas import ObservableSchema
+from api.utils import get_json, get_jwt, jsonify_data, jsonify_errors
+
+enrich_api = Blueprint('enrich', __name__)
+
+
+get_observables = partial(get_json, schema=ObservableSchema(many=True))
+
+
+def get_judgement(observable_value, observable_type,
+                  disposition_name, valid_time):
+
+    source_uri = current_app.config['API_SOURCE_URL']\
+        .format(observable=observable_value)
+
+    return {
+        'id': f'transient:{uuid4()}',
+        'observable': {'value': observable_value, 'type': observable_type},
+        'disposition': 2,
+        'disposition_name': disposition_name,
```

Comment:
`'disposition': 2` already implies `'disposition_name': 'Malicious'` in CTIM, so probably this argument is redundant.

Reply:
I agree. Thanks!